waves/src/App.tsx

```diff
@@ -315,6 +317,7 @@ function App() {
                                     variant="main_button"
                                     isLoading={isCreatingNewSwap}
                                     disabled={isSwapButtonDisabled}
+                                    data-cy="swap-button"
```

Comment:
The `data-cy` naming is not quite consistent. I propose to always include the element type, i.e.:

- `button-create-wallet`
- `button-unlock-wallet`
- `button-swap`

We can also flip the order if that is more readable: `create-wallet-button`, etc